gunicorn.conf.py

```diff
@@ -16,6 +15,7 @@
 def start_websockify(websockify_path, target_file):
     result = subprocess.run(['pgrep', 'websockify'], stdout=subprocess.PIPE)
     if not result.stdout:
+        print("Websockify is stopped. Starting websockify.")
```

Comment:
Do you want to keep this debug print here? Does it actually print? I know sometimes Flask is weird and will only print if its to stderr

Reply:
Yes, it prints in OKD.